src/base/_typography.scss

```diff
@@ -14,4 +14,65 @@ html {
 body {
   text-size-adjust: none; /* 1 */
   word-wrap: break-word; /* 2 */
+  font-family: 'Source Sans Pro', sans-serif;
+
+  @supports (font-variation-settings: normal) {
+    font-family: 'Source Sans Pro Variable', sans-serif;
+  }
+}
+
+@font-face {
+  font-family: 'Source Sans Pro Variable';
+  src: url('../src/fonts/source-sans-pro-latin-extended/variable-regular.woff2')
+      format('woff2-variations'),
+    url('../src/fonts/source-sans-pro-latin-extended/variable-regular.woff2')
+      format('woff2');
+  font-weight: 100 900;
+  font-style: normal;
+}
+@font-face {
+  font-family: 'Source Sans Pro Variable';
+  src: url('../src/fonts/source-sans-pro-latin-extended/variable-italic.woff2')
+      format('woff2-variations'),
+    url('../src/fonts/source-sans-pro-latin-extended/variable-italic.woff2')
+      format('woff2');
+  font-weight: 100 900;
+  font-style: italic;
+}
+
+// Fallback for browsers that don't support variable fonts
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('../src/fonts/source-sans-pro-latin-extended/400.woff2')
+      format('woff2'),
+    url('../src/fonts/source-sans-pro-latin-extended/400.woff') format('woff');
+  font-weight: 400;
+}
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('../src/fonts/source-sans-pro-latin-extended/600.woff2')
+      format('woff2'),
+    url('../src/fonts/source-sans-pro-latin-extended/600.woff') format('woff');
+  font-weight: 600;
+}
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('../src/fonts/source-sans-pro-latin-extended/700.woff2')
+      format('woff2'),
+    url('../src/fonts/source-sans-pro-latin-extended/700.woff') format('woff');
+  font-weight: 700;
+}
+@font-face {
+  font-family: 'Source Sans Pro';
+  src: url('../src/fonts/source-sans-pro-latin-extended/italic.woff2')
+      format('woff2'),
+    url('../src/fonts/source-sans-pro-latin-extended/italic.woff')
+      format('woff');
+  font-style: italic;
+}
+@font-face {
+  font-family: 'Source Code Pro';
+  src: url('../src/fonts/source-code-pro-latin/400.woff2') format('woff2'),
+    url('../src/fonts/source-code-pro-latin/400.woff') format('woff');
+  font-weight: 400;
```

Comment:
Idea: Maybe we could write a Sass mixin to keep these declarations a tad more DRY?